server/src/main/java/org/elasticsearch/action/admin/indices/shards/TransportIndicesShardStoresAction.java

```diff
@@ -184,8 +187,8 @@ private class InternalAsyncFetch extends AsyncShardFetch<NodeGatewayStartedShard
 
             private final Releasable ref = refs.acquire();
 
-            InternalAsyncFetch(Logger logger, String type, ShardId shardId, String customDataPath) {
-                super(logger, type, shardId, customDataPath);
+            InternalAsyncFetch(Logger logger, String type, ShardId shardId, String customDataPath, int nodeNumber) {
```

Comment:
naming nit/suggestion

```suggestion
            InternalAsyncFetch(Logger logger, String type, ShardId shardId, String customDataPath, int expectedSize) {
```